### PR TITLE
WIP: Add flashing procedure for MA5671A

### PR DESCRIPTION
**Pull request by nanomad**

### CODE REVIEW

File: _ont/ont-huawei-ma5671a.md [hunk 1]
```diff
@@ -69,6 +66,17 @@ set nDyingGaspEnable disable
 | mtd4 | 00370000 | 00010000  | "rootfs_data" |
 | mtd5 | 00800000 | 00010000  | "image1"      |
 
+## When booting from image1
+
+| dev  | size     | erasesize | name          |
+| ---- | -------- | --------- | ------------- |
+| mtd0 | 00040000 | 00010000  | "uboot"       |
+| mtd1 | 00080000 | 00010000  | "uboot_env"   |
+| mtd2 | 00740000 | 00010000  | "image0       |
```
Comment (simonebortolin) [inline _ont/ont-huawei-ma5671a.md:75]: add " to the end

File: _ont/ont-huawei-ma5671a.md [hunk 2]
```diff
@@ -83,7 +91,27 @@ set nDyingGaspEnable disable
 - [Carlito General Setting](/ont-huawei-ma5671a-carlito)
```
Comment (simonebortolin) [inline _ont/ont-huawei-ma5671a.md:91]: the August firmware is not linked or link to a right.com.cn firmware page

File: _ont/ont-huawei-ma5671a.md [hunk 3]
```diff
@@ -49,17 +49,14 @@ Configuration: asc0=0 115200 8-N-1
 
 - [Root](/ont-huawei-ma5671a-root)
 
-##  Disabling Dying Gasp
-```sh
-set nDyingGaspEnable disable
```
Comment (ErnyTech) [inline _ont/ont-huawei-ma5671a.md:54]: Should be replaced with `fw_setenv nDyingGaspEnable 0`

**Review by ErnyTech — CHANGES_REQUESTED** [review]

**Review by nanomad — COMMENTED** [review]
Fix typo

File: _ont/ont-huawei-ma5671a.md (hunk 3 above)
Comment (nanomad) [reply _ont/ont-huawei-ma5671a.md:54]: Should be moved into the rooted section IMHO

**Review by ErnyTech — APPROVED** [review]

**Pull request merged**